test/e2e/switchable-runtime/index.test.ts

```diff
@@ -179,40 +179,6 @@ describe('Switchable runtime', () => {
         response = await fetchViaHTTP(context.appPort, '/api/edge')
         text = await response.text()
         expect(text).toMatch(/Returned by Edge API Route .+\/api\/edge/)
-
```

Comment:
most of the nodejs/edge routes tests are also covered in other app-dir tests already, this is a pretty legacy test that can be deleted 👍